src/core_plugins/elasticsearch/lib/__tests__/kibana_index_mappings.js

```diff
@@ -0,0 +1,44 @@
+import expect from 'expect.js';
+import sinon from 'sinon';
+import { has } from 'lodash';
+import { KibanaMappings } from '../kibana_index_mappings';
+
+
+describe('plugins/elasticsearch', function () {
+  describe('lib/kibana_index_mappings', function () {
+    let kibanaMappings;
+    const plugin = {
+      status: {
+        red: sinon.stub()
+      }
+    };
+    beforeEach(function () {
+      kibanaMappings = new KibanaMappings(plugin);
+    });
+
+    it('provides default mappings', function () {
+      expect(kibanaMappings.getCombined()).to.an('object');
```

Comment:
It doesn't seem to matter, but normally there'd be a `be` between `to` and `an`.